### ISSUE TITLE
model dependency on 'neuralcoref'?

### ISSUE THREAD

**Issue by alloplastic** [huggingface/neuralcoref#51]
The models appear to have a dependency on a "neuralcoref" module.  Should this be installed in addition to a model such as "en-coref-lg"?

I ask due to the following error I'm seeing when running nueralcoref inside a Docker container.

In the Dockerfile, I run:

`RUN pip install https://github.com/huggingface/neuralcoref-models/releases/download/en_coref_lg-3.0.0/en_coref_lg-3.0.0.tar.gz`

Then, in the Python service, I initialize coref like this:

```
     import spacy
     coref = spacy.load('en_coref_lg')
```

 Which leads to the following error on the 2nd line above:

```
nlu-service         | INFO:root:Loading spacy model, wait for confirmation before using
nlu-service         | INFO:root:Loaded spacy model
nlu-service         | Traceback (most recent call last):
nlu-service         |   File "/app/main.py", line 58, in <module>
nlu-service         |     coref = spacy.load('en_coref_lg')
nlu-service         |   File "/usr/local/lib/python3.6/site-packages/spacy/__init__.py", line 15, in load
nlu-service         |     return util.load_model(name, **overrides)
nlu-service         |   File "/usr/local/lib/python3.6/site-packages/spacy/util.py", line 114, in load_model
nlu-service         |     return load_model_from_package(name, **overrides)
nlu-service         |   File "/usr/local/lib/python3.6/site-packages/spacy/util.py", line 134, in load_model_from_package
nlu-service         |     cls = importlib.import_module(name)
nlu-service         |   File "/usr/local/lib/python3.6/importlib/__init__.py", line 126, in import_module
nlu-service         |     return _bootstrap._gcd_import(name[level:], package, level)
nlu-service         |   File "<frozen importlib._bootstrap>", line 994, in _gcd_import
nlu-service         |   File "<frozen importlib._bootstrap>", line 971, in _find_and_load
nlu-service         |   File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
nlu-service         |   File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
nlu-service         |   File "<frozen importlib._bootstrap_external>", line 678, in exec_module
nlu-service         |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
nlu-service         |   File "/usr/local/lib/python3.6/site-packages/en_coref_lg/__init__.py", line 6, in <module>
nlu-service         |     import neuralcoref 
nlu-service         | ModuleNotFoundError: No module named 'neuralcoref'
```

In other words, Python finds the en_coref_lg model, but the model is looking for something that isn't there.  Have I missed a step?

Thanks in advance for any help you can provide...

**Comment by theSage21** [comment]
This might be related to https://github.com/huggingface/neuralcoref/pull/44

**Comment by thomwolf** [comment]
Yes, I am on it now!

**Comment by thomwolf** [comment]
I am uploading updated versions of the models (`en_coref_sm` is up, the others are currently being uploaded) in which I fixed the install scripts.
They should work with the original readme instructions.
You can try to download and install them (like before: `pip install MODEL_URL`).

**Comment by alloplastic** [comment]
Thanks!  I verified that I am able to load the "en_coref_sm" model successfully.

Unfortunately, I think one or both of the other models may not have uploaded correctly.  I get the following in a browser when I enter the URL for the large model (https://github.com/huggingface/neuralcoref-models/releases/download/en_coref_lg-3.0.0/en_coref_lg-3.0.0.tar.gz).

```
<Error>
  <Code>NoSuchKey</Code>
  <Message>The specified key does not exist.</Message>
  <Key>134695006/d5639540-7311-11e8-8c34-5c74b6924099</Key>
  <RequestId>D4957F53387708AB</RequestId>
  <HostId>
    koDcmCwivn5l/T9yZ7Ceo3xPb58pcTSizDO/J1GcqVfahWV9h+F2xsrw6py8GOHZ
  </HostId>
</Error>
```

(Not a blocking issue for me at the moment, though.)

Thanks again for the help...
